api/include/mfxstructures.h

```diff
@@ -2394,7 +2394,7 @@ MFX_PACK_BEGIN_USUAL_STRUCT()
 typedef struct {
     mfxExtBuffer Header;
 
-    mfxU16 Flags;           /* FilmGrainFlags */
+    mfxU16 FilmGrainFlags;  /* FilmGrainFlags */
```

Comment:
@onabiull do you see any issues on renaming fields of structs which are still under ```VERSION_NEXT```?

Reply:
No :) Fields under that define considered volatile. They may be even dropped without any warning.